subsys/bluetooth/audio/has.c

```diff
@@ -1122,12 +1132,23 @@ int bt_has_preset_name_change(uint8_t index, const char *name)
 }
 #endif /* CONFIG_BT_HAS_PRESET_SUPPORT */
 
-static int has_init(const struct device *dev)
+int bt_has_register(const struct bt_has_register_param *param)
 {
-	ARG_UNUSED(dev);
+	LOG_DBG("Trying to start HAS service...");
+	int err;
+
+	CHECKIF(!param) {
+		LOG_DBG("NULL params pointer");
+		return -EINVAL;
+	}
+
+	CHECKIF(param->hearing_aid_type > 2) {
+		LOG_DBG("Invalid Hearing Aid Type 0x%x", param->hearing_aid_type);
```

Comment:
BT_DBG

Reply:
@fredrikdanebjer The BT_DBG/WARN/ERR logging functions has been removed by https://github.com/zephyrproject-rtos/zephyr/pull/51897 :)